src/test/run-make/raw-dylib-alt-calling-convention/Makefile

```diff
@@ -0,0 +1,18 @@
+# Test the behavior of #[link(.., kind = "raw-dylib")] with alternative calling conventions.
+
+# only-i686-pc-windows-msvc
```

Comment:
I suppose this should be testable on all MSVC targets since we are normalizing stdcall/fastcall/etc to C on everything except for `i686`?
(This kind of depends on decisions in https://github.com/rust-lang/rust/pull/86231 again.)

Reply:
Well, the test passes when run in x86_64-pc-windows-msvc, though I'd like additional clarity on what we want to do with the various calling conventions on non-i686(/i586) platforms before relaxing the `only` header command.

Reply:
Ok, let's leave this as is in accordance with https://github.com/rust-lang/rust/pull/86419#discussion_r660780299.